dashboard/app/controllers/levels_controller.rb

```diff
@@ -132,7 +132,7 @@ def show
 
     view_options(
       full_width: true,
-      small_footer: @game.uses_small_footer? || @level.enable_scrolling?,
+      small_footer: @game&.uses_small_footer? || @level.enable_scrolling?,
       has_i18n: @game.has_i18n?,
```

Comment:
this may be a bit of whack-a-mole, but I would suggest also doing `@game.` -> `@game&.` in the other places in these methods and then calling it good.

Reply:
using a safe nav in every place is the best way to do this, since uses_small_footer isn't present in every situation.